website/registrations/README.md

```diff
@@ -49,9 +52,16 @@ If the automatically created username is not unique, the board can override it b
     - They indicate whether they still are 'member' or want to become a 'benefactor'.
     - If 'benefactor', the user picks a contribution amount (with minimum).
     - A 'member' with a current 'year' membership can upgrade to a 'study' membership. There is a discount for this.
-    - An expired 'member' can get a new 'study' membership. This is currently not discounted, but this may change depending on the Rules and Regulations.
+    - An expired 'member' can get a new 'study' membership.
```

Comment:
Because this changed i believe now you can still get discounted study long even if it's recently been expired but im not sure @DeD1rk should know this